server/routes/user.js

```diff
@@ -1,95 +1,131 @@
 const express = require("express");
 const router = express.Router();
 const bcrypt = require("bcrypt");
-var Parse = require('parse/node');
+var Parse = require("parse/node");
 
-Parse.initialize("FYm3VuO0u2fZehFsYlcKZSuloKs5Bf75EBJUhGf7","CWltjX0VqknaSsdHrgoTjVWhCDGrUGINIO6dEqUB"); //PASTE HERE YOUR Back4App APPLICATION ID AND YOUR JavaScript KEY
-Parse.serverURL = 'https://parseapi.back4app.com/'
+Parse.initialize(
+  "FYm3VuO0u2fZehFsYlcKZSuloKs5Bf75EBJUhGf7",
```

Comment:
Something I'd watch out for is to not leak your credentials (otherwise randos on the internet can stela them). 

Not something to fix now if you're prototyping but when you have the chance look up how to use gitignore and add your certificates.